iati/core/default.py

```diff
@@ -133,18 +134,42 @@ def codelists(version=None):
     """Locate the default Codelists for the specified version of the Standard.
 
     Args:
-        version (str): The version of the Standard to return the Codelists for. Defaults to None. This means that the latest version of the Codelist is returned.
-
-    Raises:
-        ValueError: When a specified version is not a valid version of the IATI Standard.
+    version (str): The version of the Standard to return the Codelists for. Defaults to None. This means that the latest version of the Codelist is returned.
 
     Returns:
-        dict: A dictionary containing all the Codelists at the specified version of the Standard. All Non-Embedded Codelists are included. Keys are Codelist names. Values are iati.core.Codelist() instances.
+    dict: A dictionary containing all the Codelists at the specified version of the Standard. All Non-Embedded Codelists are included. Keys are Codelist names. Values are iati.core.Codelist() instances.
 
     """
     return _codelists(version)
 
 
+def ruleset(version=None):
+    """Locate the default Ruleset for the specified version of the Standard.
+
+    Args:
+        version (str): The version of the Standard to return the Ruleset for. Defaults to None. This means that the latest version of the Ruleset is returned.
+
+    Returns:
+        iati.core.ruleset.Ruleset: The default Ruleset for the specified version of the Standard.
+
+    Raises:
+        ValueError: When a specified version is not a valid version of the IATI Standard.
+
+    """
+    path = iati.core.resources.get_ruleset_path(iati.core.resources.FILE_RULESET_STANDARD_NAME, version)
+    ruleset_str = iati.core.resources.load_as_string(path)
+
+    return iati.core.Ruleset(ruleset_str)
+
+
+def ruleset_schema(version=None):
```

Comment:
Is this something to go in `rulesets.py` since there isn't really a thing as a non-default Ruleset Schema?